frontend/deploy/travella-frontend.ingress.yml.tpl

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ PROJECT_NAME }}
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: {{ PROJECT_NAME }}
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: {{ INGRESS_STATIC_IP_NAME }}
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /
```

Comment:
I think you missed out the `*`, I believe going by the session we had it should be `/*`